### ISSUE TITLE
GLaDOS Voice Line Trigger

### ISSUE THREAD

**Issue by janerik20** [BEEmod/BEE2.4#2060]
### Description of the feature

I would like to suggest the following feature:
There are voice line triggers that can trigger voice lines from GLaDOS. These could possibly be configured in BeeMod (select specific voice line) and then built into Portal 2 as a trigger. If it is possible: I would like to make everything into an item that I have enough room for other things in my Portal 2 items inventory.

### Why should this be implemented

This could make the level more lively. If something specific happens in the level, a voice line could make the part more exciting.

**Comment by CompyCube** [comment]
Good idea and it could possibly be made by a simple trigger linking to a actor in the game. It technically already exists too via the "mid" and "response" categories in most voice options.

**Comment by unfunny1** [comment]
there's a ucp for that, forgot the name of it tho

**Comment by TeamSpen210** [comment]
The main problem with these sorts of things is that currently, you're able to arbitrarily swap style or characters, and everything will still work. If we do add a voiceline trigger, it's really not clear what should happen if you switch to another character with those present in the map. Should it be limited to a preset list that corresponds to each other? Or just accept that it will produce nonsense if you switch.
